website/docs/reference/api/legacy/unleash/admin/metrics.md

```diff
@@ -152,7 +157,13 @@ This endpoint gives insight into details about a client application, such as ins
 
 ### Seen applications {#seen-applications}
 
-`GET http://unleash.host.com/api/admin/seen-apps`
+:::caution Deprecation notice
+
+This endpoint has been deprecated
```

Comment:
Same as above: what do they use instead?t

```suggestion
This endpoint has been deprecated. Use ... instead.
```

Reply:
There's no replacement AFAIK

Reply:
And they're returning an empty response with 410 response code:
https://github.com/Unleash/unleash/blob/0b18491237686be627edc52fc953d37e9624371d/src/lib/routes/admin-api/metrics.ts#L120-L126